app/src/main/java/de/pscom/pietsmiet/MainActivity.java

```diff
@@ -196,4 +220,19 @@ protected void onPause() {
             PostCache.setPosts(cardManager.getAllCardItems());
         }
     }
+
+    private void updateData()
+    {
+        if(cardManager == null)
+            return;
+        List<Post> cards = cardManager.getAllCardItems();
+        // (optional?)
+        PostCache.setPosts(cards);
+        cards.clear();
+        cards.addAll(PostCache.getPosts());
```

Comment:
Crash an dieser Stelle, weil der CacheManager hier null ist.

java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String[] de.pscom.pietsmiet.io.caching.CacheManager.readLines(java.lang.String)' on a null object reference
                                                                      at de.pscom.pietsmiet.io.caching.PostCache.getPosts(PostCache.java:43)
                                                                      at de.pscom.pietsmiet.MainActivity.updateData(MainActivity.java:226)
                                                                      at de.pscom.pietsmiet.MainActivity.access$000(MainActivity.java:29)
                                                                      at de.pscom.pietsmiet.MainActivity$1.onRefresh(MainActivity.java:62)